daemon/graphdriver/overlay2/overlay.go

```diff
@@ -171,6 +173,13 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 			}
 		}
 	}
+	supportsDType, err := fsutils.SupportsDType(home)
```

Comment:
OK, you use `home` which is set to `/var/lib/docker/overlay2` by default and the directory might not exist yet. I guess a call to `filepath.Dir(home)` (same as in `graphdriver.GetFSMagic()`) is still needed either here or directly in `fsutils.SupportsDType()`.